common/services/catalogue/params.js

```diff
@@ -70,17 +70,19 @@ function buildDeserialiser<T>(deserialisers: Deserialisers<T>) {
 type SerialisedParams = { [key: string]: ?string };
 function buildSerialiser<T>(
   serialisers: Serialisers<T>,
-  queryStringParameterMapping: QueryStringParameterMapping
+  queryStringParameterMapping: QueryStringParameterMapping,
+  isForUrl?: boolean
```

Comment:
Do we think we should have a better name for this – it's to distinguish between URLs for the app and the API, right? would `isForApp` or `isForAppUrl` better? (I'm not sure!)

Reply:
I think though that's the point of being able to write multiple serialisers